src/vsr/replica.zig

```diff
@@ -424,6 +433,13 @@ pub fn ReplicaType(
                     self.transition_to_recovering_head();
                 }
             }
+
+            assert(
+                self.status == .recovering and self.replica_count == 1 or
```

Comment:
Can you add parens around the `and` expression? Just to make the precedence obvious.